awx/ui_next/src/components/Lookup/CredentialLookup.test.jsx

```diff
@@ -90,7 +89,8 @@ describe('CredentialLookup', () => {
   });
 
   test('should auto-select credential when only one available and autoPopulate prop is true', async () => {
-    CredentialsAPI.read.mockReturnValue({
+    jest.resetAllMocks();
```

Comment:
doing this at the beginning of a test feels icky. Should this be in `afterEach` instead?

Reply:
Yeah, it's weird.

On closer inspection this code appears to be overriding the mock data used in the `beforeEach` and so relocating the `resetAllMocks` to an `afterEach` (or to the end of any test case) won't get the test passing again.

So, to clean up that rogue `resetAllMocks` I just moved the test case out to it's own test suite.

![Screenshot from 2021-04-19 13-24-51](https://user-images.githubusercontent.com/9753817/115277766-a8e51200-a112-11eb-90ac-afb8be3b0776.png)

Reply:
yeah, this would be worth getting to the bottom of at some point, but I wouldn't hold up this merge for it